src/elements/embed/embedComponents/Preview.tsx

```diff
@@ -1,17 +1,43 @@
 import { css } from "@emotion/react";
+import { useRef, useState } from "react";
 import { Label } from "../../../editorial-source-components/Label";
 import { unescapeHtml } from "./embedUtils";
 
-const iframe = css`
-  background-color: white;
-  width: 100%;
-`;
-
 export const Preview = ({ html }: { html: string }) => {
```

Comment:
Can we use React.memo to prevent this component rerendering if `html` doesn't change? Aware that unescapeHtml might be doing some 💪 work 💪 here!